modules/repository-s3/src/test/java/org/elasticsearch/repositories/s3/S3RepositoryTests.java

```diff
@@ -88,10 +88,14 @@ public void testInvalidChunkBufferSizeSettings() {
     }
 
     private Settings bufferAndChunkSettings(long buffer, long chunk) {
-        return Settings.builder()
-            .put(S3Repository.BUFFER_SIZE_SETTING.getKey(), new ByteSizeValue(buffer, ByteSizeUnit.MB).getStringRep())
-            .put(S3Repository.CHUNK_SIZE_SETTING.getKey(), new ByteSizeValue(chunk, ByteSizeUnit.MB).getStringRep())
-            .build();
+        return defaultRepoSettings().put(
+            S3Repository.BUFFER_SIZE_SETTING.getKey(),
+            new ByteSizeValue(buffer, ByteSizeUnit.MB).getStringRep()
+        ).put(S3Repository.CHUNK_SIZE_SETTING.getKey(), new ByteSizeValue(chunk, ByteSizeUnit.MB).getStringRep()).build();
+    }
+
+    private Settings.Builder defaultRepoSettings() {
```

Comment:
nit: likewise here. Indeed extracting this method kinda messes up the otherwise-quite-nicely-formatted chain of `.put` calls in `bufferAndChunkSettings()`.